src/main/java/org/traccar/protocol/WatchProtocolDecoder.java

```diff
@@ -89,6 +89,8 @@ private String decodeAlarm(int status) {
             return Position.ALARM_GEOFENCE_EXIT;
         } else if (BitUtil.check(status, 2)) {
             return Position.ALARM_GEOFENCE_ENTER;
+        } else if (BitUtil.check(status, 14)) {
+            return Position.ALARM_POWER_CUT; // some 2G devices raise this alarm when the device is unplugged from its charger
```

Comment:
Comment is unnecessary here. Please remove.

Reply:
ok